server/testdb/postgres.go

```diff
@@ -81,11 +81,7 @@ func (td *postgresDB) ServerID() (id string, isNew bool, err error) {
 
 	// no id, let's creat it
 	isNew = true
-	id, err = machineid.ProtectedID("tracetest")
-	if err != nil {
-		err = fmt.Errorf("could not get machineID: %w", err)
-		return
-	}
+	id = config.GetMachineID()
```

Comment:
Does make sense try to use [this unit test](https://github.com/kubeshop/tracetest/pull/1796/files#diff-c4d616c3d0d491af1cba1ba8028c52a51f97ffbb2461732204bb5936e8f0583b) here?